backend/api/v2beta1/experiment.proto

```diff
@@ -15,7 +15,7 @@
 syntax = "proto3";
 
 option go_package = "github.com/kubeflow/pipelines/backend/api/v2beta1/go_client";
-package v2beta1;
+package kubeflow.pipelines.backend.api.v2beta1;
```

Comment:
Hi @Linchin. This should be already in the [master branch](https://github.com/kubeflow/pipelines/blob/master/backend/api/v2beta1/experiment.proto#L18) after [this change](https://github.com/kubeflow/pipelines/pull/8533/files#diff-ca1b12d8b07102930c2c7d54e2d934b84f2075b00500ba25fea2f7a67e4fe927R18).